src/components/MdTable/MdTable.vue

```diff
@@ -45,6 +45,9 @@
       <slot name="md-table-pagination" />
     </md-content>
 
+    <slot name="md-table-loading"></slot>
+    <md-progress-bar md-mode="indeterminate" v-if="!$scopedSlots['md-table-loading'] && asyncLoading" />
+
```

Comment:
How about this?

```html
<slot name="md-table-loading" v-if="asyncLoading">
  <md-progress-bar md-mode="indeterminate" />
</slot>
```

Reply:
Thanks for helping me out with this! Not extremely familiar with slots myself.